### PR TITLE
Add custom String() implementations to protos.

### PR DESCRIPTION
**Pull request by PFedak**
Give pfs protos commonly mentioned in error strings and other
user-facing messages use new custom String implementations which match
our preferred representations so that they are handled consistently.

### CODE REVIEW

**Comment by PFedak** [comment]
This should make working with our protos a little easier, and address the first part of #6344, independent of whether we eliminate the user suffix entirely

File: src/pfs/pfs.proto [hunk 1]
```diff
@@ -14,11 +14,13 @@ import "auth/auth.proto";
 ////  PFS Data structures (stored in etcd)
 
 message Repo {
+  option (gogoproto.goproto_stringer) = false;
```
Comment (Tryneus) [inline src/pfs/pfs.proto:17]: the other options we use put these in `[]` - do we know why that is?  maybe just do the same thing here for consistency?

**Review by Tryneus — APPROVED** [review]
LGTM

File: src/pfs/pfs.proto (hunk 1 above)
Comment (PFedak) [reply src/pfs/pfs.proto:17]: I think that might be special syntax for the ones that apply to specific fields, but I can try to figure that out

Comment (PFedak) [reply src/pfs/pfs.proto:17]: Yeah, seems to just be for tags

**Pull request merged**